backend_addon/{{ cookiecutter.__folder_name }}/README.md

```diff
@@ -19,6 +19,36 @@ And to create the Plone site:
 make create_site
 ```
 
+## Add features using `plonecli` or `bobtemplates.plone`
+
+This package provides markers as strings (`<!-- extra stuff goes here -->`) that are compatible with [`plonecli`](https://github.com/plone/plonecli) and [`bobtemplates.plone`](https://github.com/plone/bobtemplates.plone).
+These markers act as hooks to add all kinds of subtemplates, including behaviors, control panels, upgrade steps, or other subtemplates from `plonecli`.
+
+To use them, you need to use `bobtemplates.plone` version X.X.X or later, and run the following command.
```

Comment:
We need to update the version number here. The current version of `bobtemplates.plone` is 6.3.3. Is this feature already in a release or a future one?

Reply:
No yet, there's a pending PR on bobtemplates.plone to be able to use these changes.

Reply:
@erral Any update?

Reply:
The PR was merged, but the release is still missing.